src/audio/host-zephyr.c

```diff
@@ -842,11 +842,21 @@ int host_common_params(struct host_data *hd, struct comp_dev *dev,
 		hd->dma_buffer_size = audio_stream_get_size(&hd->dma_buffer->stream);
 	}
 
+	dma_block_cfg = rballoc(SOF_MEM_FLAG_COHERENT,
+				SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
+				sizeof(struct dma_block_config));
```

Comment:
please use `rzalloc()` and remove `memset()` below

Reply:
done